README.md

```diff
@@ -95,6 +95,25 @@ The returned event payload is the same as the input.
         "message": "..."
     }
 
+### ReactionAdded
+    {
+
+        "type":"reaction_added", 
+        "user":"...",  //user that adds the reaction
+        "itemUser":"...",  //user that writes the message, file etc.
+        "item": {
+                "type" :"message", 
+                "channel" :"...", 
+                "file" :"", 
+                "fileComment" :"", 
+                "timestamp" :"..."
+                }, 
+        "reaction" :"...", //value of the reaction
+        "EventTimestamp" :"..." 
```

Comment:
This is incorrect. This has to match `reactionAddedEvent` type e.g. there is no such field as `item`.

I've updated type in suggestions so this will have to be updated as well

```suggestion
        "type":"...",
        "user":"...",  //user that adds the reaction
        "itemUser":"...",  //user that writes the message, file etc.
        "item": {
                "type" :"message", 
                "channel" :"...", 
                "timestamp" :"..."
                }, 
        "reaction" :"...", //value of the reaction
        "eventTimestamp" :"..." 
```